QEfficient/utils/_utils.py

```diff
@@ -5,6 +5,7 @@
 #
 # -----------------------------------------------------------------------------
 
+import hashlib
```

Comment:
Do we need this package? or its preinstalled in unix?

Reply:
its a built-in library in Python
https://docs.python.org/3.8/library/hashlib.html